### ISSUE TITLE
RCO causing long load times

### ISSUE THREAD

**Issue by Thelolguy1** [L8X/Roblox-Client-Optimizer#32]
Hello,

We've noticed extremely long load times on the game FRONTLINES (and possibly other games) in Vinegar. We suspect that [this commit](https://github.com/L8X/Roblox-Client-Optimizer/commit/fa1f5f49ce944ef8b178cfcfacb7393fd79ce7a3) is causing trouble. 

Thanks

**Comment by Thelolguy1** [comment]
Yep, I just disabled the flags listed on that commit and load times are fine again

**Comment by apprehensions** [comment]
Please have some form of test suite to use with games, to also know the performance difference.

**Comment by Thelolguy1** [comment]
![image](https://user-images.githubusercontent.com/18731514/229274089-93d0c8c9-5944-4d69-8e31-11c1d03f4865.png)
Additionally, disabling those flags has led to this...

**Comment by apprehensions** [comment]
It still takes like, 2 seconds from 20, so i guess its an improvement??
